src/main/java/org/folio/sidecar/utils/RoutingUtils.java

```diff
@@ -143,6 +144,11 @@ public static boolean isSystemRequest(RoutingContext rc) {
     return SYS_INTERFACE_TYPE.equals(scRoutingEntry.getInterfaceType());
   }
 
+  public static boolean isTimerRequest(RoutingContext rc) {
```

Comment:
it looks like this condition is always used with negate.
Maybe rename it to `isNotTimerRequest` and add negation in implementation?

Reply:
This will be contrary to current implementation where all methods are isSomething, not isNotSomething

Reply:
P.S. "this condition is always used with negate"
At the moment, yes - because it's not used that much. But in the future it can potentially be used without negation

Reply:
P.P.S. Another example is hasSystemAccessToken - it's also used only with negation, but it's still hasSystemAccessToken, not hasNoSystemAccessToken